frontend/src/services/loginService.ts

```diff
@@ -0,0 +1,14 @@
+import { useAuthStore } from '../store/authStore.ts';
+import { ACCESS_TOKEN_KEY } from '@/constants/auth.ts';
+
+export const login = (token: string) => {
+  localStorage.setItem(ACCESS_TOKEN_KEY, token); // 토큰을 브라우저에 저장
+  const setToken = useAuthStore.getState().setToken; // 전역 상태 갱신
+  setToken(token);
+};
+
+export const logout = () => {
+  localStorage.removeItem(ACCESS_TOKEN_KEY);
+  const setToken = useAuthStore.getState().setToken;
+  setToken(null);
+};
```

Comment:
이거 제가 마인드맵 코드 작업할 때처럼 Auth 관련 함수들을 객체 형태로 묶는 건 어떤가요?

실제로 아래처럼 구성하면 스타일도 일관되고, 나중에 인증 관련 기능을 추가할 때도 접근이 편하더라고요! 저도 그래서 마인드맵 작업할 때 이런 식으로 정리했었어요 😄
다운님도 인증 기능 확장할 때 훨씬 편하실 것 같아서 제안드려봅니다!

확인해주시고 의견 편하게 주세요!

```suggestion
export const authService = {
  login: (token: string) => {
    localStorage.setItem(ACCESS_TOKEN_KEY, token); // 토큰을 브라우저에 저장
    const setToken = useAuthStore.getState().setToken; // 전역 상태 갱신
    setToken(token);
  },
  
  logout: () => {
    localStorage.removeItem(ACCESS_TOKEN_KEY);
    const setToken = useAuthStore.getState().setToken;
    setToken(null);
  }
};
```

Reply:
왕 너무 좋은 제안입니다...ㅜㅜ 반영하여 수정했습니다 감사합니다!